Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Use an official Python runtime as a parent image
+FROM python:3.11-slim-buster
+
+# Set the working directory in the container to /app
+WORKDIR /app
+
+# Install git
+RUN apt-get update && \
+    apt-get upgrade -y
+
+# Install Poetry
+RUN pip install poetry
```

Comment:
**Suggestion:** Pin the versions of the packages you are installing to ensure reproducible builds.
```suggestion
RUN pip install poetry==1.1.13  # Replace with the desired version
```